engine/launcher/saveinterface.cpp

```diff
@@ -413,538 +436,581 @@ int main( int   argc,
+        gtk_widget_show(newb);
+        gtk_container_add(GTK_CONTAINER(GTK_FILE_SELECTION(filew)->button_area), newb);
+        gtk_signal_connect(GTK_OBJECT(newb), "clicked", (GtkSignalFunc) copynormal_conf, filew);
+    }
+    gtk_signal_connect(GTK_OBJECT(GTK_FILE_SELECTION(filew)->ok_button),
+                       "clicked", (GtkSignalFunc) func, filew);
+    GTK_FILE_SELECTION(filew)->help_button = gtk_button_new_with_label("Help");
     GTK_WIDGET_SET_FLAGS(GTK_FILE_SELECTION(filew)->help_button, GTK_CAN_DEFAULT);
-    gtk_signal_connect (GTK_OBJECT (GTK_FILE_SELECTION (filew)->help_button),
-                        "clicked", (GtkSignalFunc) help_func, (void*)i );
+    gtk_signal_connect(GTK_OBJECT(GTK_FILE_SELECTION(filew)->help_button),
+                       "clicked", (GtkSignalFunc) help_func, (void *) i);
 
-    gtk_signal_connect_object (GTK_OBJECT (GTK_FILE_SELECTION
-                                            (filew)->cancel_button),
-                               "clicked", (GtkSignalFunc) gtk_widget_destroy,
-                               GTK_OBJECT (filew));
+    gtk_signal_connect_object(GTK_OBJECT(GTK_FILE_SELECTION
+                                                 (filew)->cancel_button),
+                              "clicked", (GtkSignalFunc) gtk_widget_destroy,
+                              GTK_OBJECT(filew));
 //    if (default_thing[0]!='\0') {
 //      gtk_file_selection_complete (GTK_FILE_SELECTION(filew), default_thing);
 //    }else {
-      gtk_file_selection_set_filename (GTK_FILE_SELECTION(filew), default_thing);
+    gtk_file_selection_set_filename(GTK_FILE_SELECTION(filew), default_thing);
 //    }
-    GtkWidget *lbl=gtk_label_new(otherstr);
-    gtk_container_add(GTK_CONTAINER(GTK_FILE_SELECTION(filew)->action_area),lbl);
+    GtkWidget *lbl = gtk_label_new(otherstr);
+    gtk_container_add(GTK_CONTAINER(GTK_FILE_SELECTION(filew)->action_area), lbl);
     gtk_widget_show(lbl);
-    GtkWidget *box=gtk_label_new("");
-    gtk_container_add (GTK_CONTAINER (GTK_FILE_SELECTION(filew)->button_area),box);
-    gtk_box_pack_end (GTK_BOX (GTK_FILE_SELECTION(filew)->ok_button->parent),GTK_FILE_SELECTION (filew)->help_button, FALSE, TRUE, 40);
-	if (!usenormalbuttons)
-		gtk_box_pack_end(GTK_BOX(GTK_FILE_SELECTION(filew)->main_vbox),GTK_FILE_SELECTION(filew)->button_area,FALSE,TRUE,0);
+    GtkWidget *box = gtk_label_new("");
+    gtk_container_add(GTK_CONTAINER(GTK_FILE_SELECTION(filew)->button_area), box);
+    gtk_box_pack_end(GTK_BOX(GTK_FILE_SELECTION(filew)->ok_button->parent),
+                     GTK_FILE_SELECTION(filew)->help_button,
+                     FALSE,
+                     TRUE,
+                     40);
+    if (!usenormalbuttons) {
+        gtk_box_pack_end(GTK_BOX(GTK_FILE_SELECTION(filew)->main_vbox),
+                         GTK_FILE_SELECTION(filew)->button_area,
+                         FALSE,
+                         TRUE,
+                         0);
```

Comment:
I've come to like this from Python; again - it would be good to be consistent - if we're going to drop the parameters this way then we should drop them all, including the first:
```C
someFunc(
  param1,
  param2,
  param3,
)
```
It takes one more line, but saves indentation spaces, as well as standardizes the indentation space since it's not based on the function name

Reply:
OK, I'll see what I can do.

Reply:
Moving the first parameter onto a second line doesn't seem to be an option in CMake's formatter. However, I was able to standardize the indentation at, I believe, 8 spaces for line continuations.

Reply:
yeah, unfortunately getting code formatters to work properly is the hardest thing in C/C++ for some reason. May be I'll have to try and write my own sometime because the state of the current ones is rather ridiculous especially since every other language seems to not have that kind of issue. aindent and GNU indent seem to be among the better ones that way, but even then... (rant over)

Reply:
Well, will this suffice for now?

Reply:
yes. I'm flexible.